docker.go

```diff
@@ -112,6 +112,9 @@ type (
 		BaseImageRegistry string  // Docker registry to pull base image
 		BaseImageUsername string  // Docker registry username to pull base image
 		BaseImagePassword string  // Docker registry password to pull base image
+		PushOnly          bool    // Push only mode, skips build process
+		SourceTarPath     string  // Path to Docker image tar file to load and push
+		TarPath           string  // Path to save Docker image as tar file
```

Comment:
DestinationTarPath ?

Reply:
will add it here https://github.com/drone-plugins/drone-buildx/blob/6d4435fe34fe14a8ec3e8730e1b3a7a7e06e9309/app.go#L431